lib/synsbasen_api/client.rb

```diff
@@ -31,10 +33,10 @@ def connection
       # @param body [Hash] Request body.
       # @return [ApiResponse] An instance of `ApiResponse` containing the API response.
       # @raise [ClientError, ServerError] Raised for client or server errors.
-      def get(path, params: {}, body: {})
+      def get(path, params: {}, body: {}, expand: [])
         response = connection.get(path) do |req|
-          req.params = params
-          req.body = body unless body.empty?
+          req.params = { **params, expand: expand }.compact_blank
+          req.body = body.to_json unless body.empty?
```

Comment:
Does it make sense to have a body in a get request?

Reply:
No, I don't think so. My latest commit removes it.